mvrx-rxjava2/src/test/kotlin/com/airbnb/mvrx/DummyJavaFileTest.java

```diff
@@ -0,0 +1,7 @@
+package com.airbnb.mvrx;
+
+/**
+ * This dummy java file is needed to make Buck and Buck Project happy.
```

Comment:
we don't use buck, why is this included?

Reply:
@elihart It was copied over from the mvrx module. I can definitely delete it.